src/allmydata/test/web/test_web.py

```diff
@@ -120,17 +122,13 @@ def get_status(self):
         return self.status
 
 
-class FakeStatusItem(object):
-    def __init__(self, p, history):
-        self.relpath_u = p
-        self.history = history
-        import mock
-        self.progress = mock.Mock()
-        self.progress.progress = 100.0
-
-    def status_history(self):
-        return self.history
-
+def create_test_queued_item(relpath_u, history=[]):
+    progress = mock.Mock()
+    progress.progress = 100.0
+    item = QueuedItem(relpath_u, progress, 1234)
+    for st, ts in history:
```

Comment:
Separately, st is probably an okay abbreviation for status and ts is probably an okay abbreviation for timestamp.  Right next to each other - `st, ts` - I think they suffer a little bit (due to the coincidence of the letters).